rocAL_pybind/examples/rocAL_api_audio_unittest.py

```diff
@@ -46,25 +48,25 @@ def plot_audio_wav(audio_tensor, idx):
     plt.savefig("OUTPUT_FOLDER/AUDIO_READER/" + str(idx) + ".png")
     plt.close()
 
-def verify_output(audio_tensor, rocal_data_path, roi_tensor, test_results, case_name):
+def verify_output(audio_tensor, rocal_data_path, roi_tensor, test_results, case_name, dimensions):
```

Comment:
Instead of getting the dimensions from the main function, can we not infer the dims from audio_tensor itself?
Can you please check @HazarathKumarM